client/src/features/projectsV2/fields/ProjectRepositoryFormField.tsx

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright 2023 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cx from "classnames";
+
+import { Controller } from "react-hook-form";
+import type { FieldValues } from "react-hook-form";
+
+import { Button, FormText, Input, Label } from "reactstrap";
+
+import type { Repository } from "../projectV2.types";
+import type { GenericProjectFormFieldProps } from "./formField.types";
+
+interface ProjectRepositoryFormFieldProps<T extends FieldValues>
+  extends GenericProjectFormFieldProps<T> {
```

Comment:
This interface does not need to be generic:

```suggestion
interface ProjectRepositoryFormFieldProps extends GenericProjectFormFieldProps<ProjectV2Repositories> {
```

Reply:
Fixed in af82cda0cfb8688eb6000b8934c8115ac37016fa